### PR TITLE
ulogd: init at 2.0.7

### PR DESCRIPTION
**Pull request by xvuko**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Review by aanderse — DISMISSED** [review]
I took a look at your module and had a few suggestions. I hope you find them helpful. If you have any questions or need any help don't hesitate to ask.

**Comment by xvuko** [comment]
@aanderse Thanks for suggestions! I have added them to my commits.

In order to build docs I added another package: linuxdoc-tools. Is it OK to add it here, or should I open another PR?

File: nixos/modules/services/logging/ulogd.nix [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+          [log1]
+          group=2
+
+          [pcap1]
+          file="/var/log/ulogd.pcap"
+          sync=1
+        '';
+        type = types.lines;
+        default = "";
+        description = "Ulogd config.";
```
[the remark this answers is not in the document]
Comment (xvuko) [reply nixos/modules/services/logging/ulogd.nix]: There is no up-to-date documentation available online so I added information about docs in ${pkgs.ulogd.doc}/share/ulogd/.

**Review by aanderse — COMMENTED** [review]
I'm not familiar with `ulogd` and I don't generally review packages, but the **module** part of this PR looks good to me :+1:

**Comment by alexvorobiev** [comment]
@xvuko I am interested in linuxdoc-tools. Did you consider separating it to another PR?

**Comment by RaitoBezarius** [comment]
Done in 207453

**Pull request closed without merging**